### PR TITLE
Update electron demo

### LINKED ISSUE #1981: Electron Demo "Click here to select a file" does not load file properly.
When clicking on the button highlighted in red, the file prompt shows but the UI does not respond to the selected file.

The functionality highlighted in green works properly.

image

Tested with the example xlsx files [here](https://file-examples.com/index.php/sample-documents-download/sample-xls-download/).

### PR DESCRIPTION
**Pull request by srijonsaha**
Continuing from #1980 fixing the commented issues. Closes #1981. Addresses #1994 as well.

### CODE REVIEW

File: demos/electron/index.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ _gaq.push(['_trackPageview']);
 
 (function() {
 	var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-	ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
+	ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'https://www') + '.google-analytics.com/ga.js';
 	var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
 })();
```
Comment (garrettluu) [inline demos/electron/index.js:30]: Not too familiar with electron, but what's the deal with doing `(function() { ... })();`? Do these lines need to be wrapped in a self-invoking function?

Comment (SheetJSDev) [reply demos/electron/index.js:30]: Honestly that entire block should be removed.  This demo started from the example at https://oss.sheetjs.com/sheetjs/ which had the GA code

**Comment by garrettluu** [comment]
Ran the demo and got an error when I tried to click the "click here to select a file" button:
```
index.js:68 Uncaught (in promise) TypeError: Cannot read property 'showOpenDialog' of undefined
    at HTMLButtonElement.handleF (index.js:68)
```

**Comment by SheetJSDev** [comment]
@garrettluu what OS / electron version are you using?

File: demos/electron/index.js (hunk 1 above)
Comment (srijonsaha) [reply demos/electron/index.js:30]: Yep these need to attached to the DOM. I guess they don't necessarily have to be wrapped in self-invoking function and could be put in the global scope of the script but the functions add some grouping to what each thing is doing. And I didn't want to change the existing structure.

**Comment by garrettluu** [comment]
> @garrettluu what OS / electron version are you using?
Electron 9.0.5, Ubuntu 20.04

**Comment by srijonsaha** [comment]
> Ran the demo and got an error when I tried to click the "click here to select a file" button:
> 
> ```
> index.js:68 Uncaught (in promise) TypeError: Cannot read property 'showOpenDialog' of undefined
>     at HTMLButtonElement.handleF (index.js:68)
> ```

@garrettluu I pushed in a fix for this. Can you try again?

**Comment by garrettluu** [comment]
> > Ran the demo and got an error when I tried to click the "click here to select a file" button:
> > ```
> > index.js:68 Uncaught (in promise) TypeError: Cannot read property 'showOpenDialog' of undefined
> >     at HTMLButtonElement.handleF (index.js:68)
> > ```
> 
> @garrettluu I pushed in a fix for this. Can you try again?

@srijonsaha Works for me now! Thanks!

**Pull request merged**